src/sourceTracking.ts

```diff
@@ -330,7 +330,7 @@ export class SourceTracking extends AsyncCreatable {
       if (options.format === 'ChangeResultWithPaths') {
         return populateFilePaths({
           elements: filteredChanges.map(remoteChangeElementToChangeResult),
-          packageDirPaths: this.project.getPackageDirectories().map((pkgDir) => pkgDir.path),
+          packageDirPaths: this.project.getPackageDirectories().map((pkgDir) => pkgDir.fullPath),
```

Comment:
Existing tests don't seem to have a problem with this change, but I'm not clear why it's necessary and why new tests aren't needed.  Is this just relative versus absolute paths?

Reply:
it's not in use by us.

https://github.com/search?q=org%3Asalesforcecli+ChangeResultWithPaths&type=code
https://github.com/search?q=org%3Aforcedotcom+ChangeResultWithPaths&type=code

I got one email about this, and I told them it wouldn't be a priority for CLI team to fix.  I started to use it for what I was building, fixed the bug, and then found a different better way, but decided to leave the bug fix in.

Reply:
and yes, the fix is "use absolute path".  the problem occurs here https://github.com/forcedotcom/source-tracking/blob/8b05aa592a0c8025d18772d236d221915d34afb0/src/shared/populateFilePaths.ts#L67 when it calls ComponentSet.fromSource with a relative path and gets a file-not-found